docs/examples/workflows/script_auto_infer.md

```diff
@@ -79,7 +79,7 @@
           command:
           - python
           image: python:3.8
-          source: "import os\nimport sys\nsys.path.append(os.getcwd())\nimport json\n\n\
-            import pickle\nwith open('/tmp/i', 'rb') as f:\n    i = pickle.load(f)\nprint(i)"
+          source: "import os\nimport sys\nsys.path.append(os.getcwd())\nimport pickle\n\
```

Comment:
Notice this changes one unrelated yaml because of the changes to `source` generation. It this a problem?